csrankings.csv

```diff
@@ -10718,10 +10725,15 @@ Kyle Winfree,Northern Arizona University,https://nau.edu/siccs/faculty/kyle-winf
 Kyo Chul Kang,POSTECH,https://cse.postech.ac.kr/kck,NOSCHOLARPAGE
 Kyogu Lee,Seoul National University,http://marg.snu.ac.kr,Fk4jQFEAAAAJ
 Kyomin Jung,Seoul National University,http://milab.snu.ac.kr/kjung/index.html,u3uMl4MAAAAJ
+<<<<<<< HEAD

```

Comment:
Please remove these merge conflict messages.

Reply:
@emeryberger My bad. Should be fixed now!